openedx/frontend-app-authoring
# refactor: Update error message while adding library team member [FC-0062]

Update the error message while adding a library team member to show the error message generated by the server

## Supporting information

- Github link: https://github.com/openedx/frontend-app-authoring/issues/1342#issuecomment-2526187323
- Internal ticket: [FAL-3978](https://tasks.opencraft.com/browse/FAL-3978)

## Testing instructions

- Select a library from the list of v2 Libraries, and open the Library Info sidebar. You need to use a user with Admin access.
- Click on Manage Access, and add a user that doesn't exist.
- Verify the new error message.
- Add a user who already is a team member.
- Verify the new error message.


## Other information

N/A
